app/helpers/meta_image_tags_helper.rb

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/ruby
+
+require 'mini_magick'
+require 'site_meta_info_defaults'
+
+
+#--------------------------
+#
+# @module MetaImageTagsHelper
+#
+# @desc Responsibility: Set meta information for images (for a page).
+# If no specific value is provided for a tag, information is looked up in
+# locale files. Falls back to site defaults if an entry is not found in
+# the locale files.
+#
+# This encapsulates all of the logic and info needed to set image tags.
+# It's complicated enough to justify pulling out into its own module.
+#
+#  NOTE: this must be called from a *View* so that Sprockets is used to get the
+#  fingerprinted asset url.  Sprockets will _only_ be called and work from a View.
+#
+# ImageMagick is used to get the image type, width, and height.
+#
+#
+# @author Ashley Engelund (ashley.engelund@gmail.com  weedySeaDragon @ github)
+# @date   2019-04-25
+#
+# @file meta_image_tags_helper.rb
+#
+#--------------------------
+module MetaImageTagsHelper
+
+  LOCALE_IMAGE_KEY = 'meta.default.image_src'
+  LOCALE_FILENAME = "#{LOCALE_IMAGE_KEY}.filename"
+  LOCALE_IMAGE_TYPE = "#{LOCALE_IMAGE_KEY}.image_type"
+  LOCALE_WIDTH = "#{LOCALE_IMAGE_KEY}.width"
+  LOCALE_HEIGHT = "#{LOCALE_IMAGE_KEY}.height"
```

Comment:
Since these are not (and presumably never will be) locale-specific, why not use constants in an initializer file?

Reply:
These are the keys to use to look things up in the locale file.  The above constants are all based on the 'main' entry for these: `LOCALE_IMAGE_KEY`  If we decide to use a different key in the locale file, then we only have to change that one "base" key and all of the others are still correct.

I'm not sure what would be gained by using an initializer file for these.  Perhaps I'm not understanding your question.

Reply:
just to recap the discussion in today's status meeting:  These values (the filename, image type, width, height) are more like _application configuration information_ and so perhaps should be stored there.

(We need to record the criteria/ differences between `ENV` variables, application initialization information (`config/initializers/...`) and `application configuration` information)